pages/index.md

```diff
@@ -1,19 +1,6 @@
-**Welcome to dYdX’s v4 public testnet! We are thrilled to share our progress so far.**
+# dYdX V4 Software Documentation
 
-Over the course of next few months, we look forward to working with you in testing out the v4 software. This will involve various aspects of the product, including:
-
-1. Core trading flows (e.g., placing an order, cancelling an order, etc.)
-2. Core user flows (e.g., deposits/withdrawals, governance, etc.)
-3. Infra-level operations (e.g., starting the network, performing upgrades or hard-forks, running indexers, etc.)
-4. Performance (e.g., network throughput, latency, etc.)
-
-For the first half of the public testnet, our focus will be on shipping and testing new features. Therefore, we plan to launch new networks in order to minimize the engineering cost to perform upgrades or migrations when shipping new features. Please expect to see network resets.
-
-For the second half of the public testnet, our focus will be on making the product more robust as we get closer to the mainnet. During this time, we plan to perform upgrades and migrations for features and bug fixes.
-
-We encourage you to ask questions, explore different features, and test limits.
-
-We appreciate your willingness to participate in the testnet and look forward to working with you!
+This site contains generic documentation the dYdX V4 software as well as information related to the dYdX Public Testnet.

```

Comment:
Very rough/generic sentence - lmk if there's suggestion for additional language

Reply:
We should get inputs from product to add some content here given that this is a landing page for the doc site. Can you follow up w Everett on this?